.pipelines/templates/run-upstream-e2e.yaml

```diff
@@ -1,8 +1,8 @@
 steps: 
   - bash: |
-      git clone -b ${UPSTREAM_BRANCH} $(upstream.repo) $(System.DefaultWorkingDirectory)/osm
+      git clone -b v${CHART_TAG} $(upstream.repo) $(System.DefaultWorkingDirectory)/osm
```

Comment:
Follow up on not passing n defaultworkingdirectory/osm

Reply:
This has something to do with how the defaultworkingdirectory is set up on ADO agents when running the CI for PRs. It doesn't seem to like it when I try to clone multiple repos to $(system.defaultworkingDirectory). So I need to add the /osm when I checkout the new repo. https://github.com/Azure/osm-azure/pull/38#discussion_r569091667